integrations.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: Integrations
+permalink: /integrations/
+redirect_from:
+  - /source-code/
+  - /downloads/
```

Comment:
These redirects are wrong. You don't want to take over these urls...

Reply:
And I delete the redirects? Or which are?

Reply:
Well, you don't want the URLs https://fuseopen.com/source-code/ or https://fuseopen.com/downloads/ to redirect to https://fuseopen.com/integrations/, I assume? I guess you copied get_started.md, which has these there to keep old links working. Just delete the "redirect_from"-list, and it should be fine.

Reply:
Yes I delete it on the new commit